### PR TITLE
Fix name of Hamstringing Strike and add effect

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

**Comment by TMunsie** [comment]
@TikaelSol Should this be implemented as a toggle that attacks at -2 if the target is hunted prey, and on hit or crit displays the correct effect to drag and drop as a note? If we're going to do this I'd rather do it all at once. Thoughts?

**Comment by simonward** [comment]
> @TikaelSol Should this be implemented as a toggle that attacks at -2 if the target is hunted prey, and on hit or crit displays the correct effect to drag and drop as a note? If we're going to do this I'd rather do it all at once. Thoughts?

Sounds good to me.

**Comment by TikaelSol** [comment]
At some point we need to start using the mark RE but I guess that can wait until we do a full refresh of Ranger since the whole thing would need revised.

File: packs/data/feat-effects.db/effect-hamstringing-strike.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "ESnzqtwSgahLcxg2",
+    "img": "systems/pf2e/icons/features/classes/debilitating-strikes.webp",
+    "name": "Effect: Hamstringing Strike",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Hamstringing Strike]{Hamstringing Strike}</p>\n<p>If you hit your prey, the creature takes a -5-foot status penalty to Speed.</p>"
```
Comment (TikaelSol) [inline packs/data/feat-effects.db/effect-hamstringing-strike.json:8]: Condense to one effect with a choiceset

**Pull request merged**